packages/semantic-model/src/fix-api-json.ts

```diff
@@ -0,0 +1,27 @@
+import { forEach, find, get } from "lodash";
+
+type LibraryFix = (libraryName: string, content: unknown) => void;
+const libraryFixes: LibraryFix[] = [addViewDefaultAggregation];
+
+export function fixLibrary(libraryName: string, fileContent: unknown): void {
+  forEach(libraryFixes, (fix) => {
+    fix(libraryName, fileContent);
+  });
+}
+
+// Exported for test purpose
+export function addViewDefaultAggregation(
+  libraryName: string,
+  content: unknown
```

Comment:
maybe I forgot something, but why is the content `unknown`? Do not we have a schema/d.ts for the api.json files?

Reply:
The fixes are done before the validation that the json matches the schema (in case we want to fix something that isn't in the schema to match the schema)